### DISCUSSION TITLE
readme file for v0.10

### DISCUSSION THREAD

**Discussion by u393385** [Azure/azure-workload-identity#641] [Q&A]
We are using v0.10. I don't find the readme file for v0.10.
I am more wondering what is the supported Kubenetes version for workload identity v0.10.
Where can I find this info?

**Comment by aramase** [comment]
@u393385 These are minor releases and there is no readme per release. Here is the matrix of k8s releases we test: https://github.com/Azure/azure-workload-identity#azure-ad-workload-identity. I would recommend using the latest releases available.

**Comment by u393385** [comment]
Ok. Thank you for the reply.



General Business
